lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -123,6 +126,8 @@ sub group_overview {
     $self->stash('only_tagged',     $only_tagged);
     $self->stash('comments',        \@comments);
     $self->stash('pinned_comments', \@pinned_comments);
+    $self->stash('latest_comment',  $latest_comment);
+    $self->stash('show_comments',   $show_comments);
```

Comment:
stash is not needed if the unique purpose is [to use it in the template](http://mojolicious.org/perldoc/Mojolicious/Plugin/DefaultHelpers#param)